README.md

```diff
@@ -35,6 +35,22 @@ Do you want to know how your shipment is going and how long it will arrive? You
 ![javascript][]
 ![react][]
 
+## Pre requirements
+
+Before to start you need have installed the following programs:
+
+1. [Node.js](https://nodejs.org/) > v14.0.0
+
+   Node.js is a runtime environment for Javascript that allows us to use the language outside of the browser, in this case we use it to configure and run the development server.
+
+2. [npm](https://www.npmjs.com/) > v6.0.0
+
+   npm is a package manager for Node.js and comes installed with it, we use it to install and manage application dependencies.
+
+3. [Git](https://git-scm.com/) > v2.0.0
+
+   Git is a version control system for code, we use it to manage the different branches of the code.
```

Comment:
What about mention using vscode, as we have support for it, also mention we need connection to internet to connect with graphql

Reply:
Yes, ok I'll write it

Reply:
The title at the line 40: " Before to start you need have installed the following programs:" don't match with the requirement "Connection to internet", since "Conection to internet" is not a program that u need to install.

But i'm agree to add "VSCode" how option requirement or program to install

Reply:
Hmm so the question is were to mention a internet connection is needed